Main/Source/gods.cpp

```diff
@@ -1349,6 +1349,26 @@ void infuscor::PrayGoodEffect()
 
 void cruentus::PrayGoodEffect()
 {
+  // Blood for the god of blood!
+  if(!RAND_4 || Relation == 1000)
+  {
+    beamdata Beam
+      (
+        0,
+        CONST_S("drowned by the blood of ") + GetName(),
+        YOURSELF,
+        0
+      );
+    lsquare* Square = PLAYER->GetLSquareUnder();
+    Square->LiquidRain(Beam, BLOOD);
```

Comment:
Got a build error here:

```
[ 72%] Building CXX object Main/CMakeFiles/ivan.dir/Source/godset.cpp.o

In file included from /home/travis/build/Attnam/ivan/Main/Source/godset.cpp:40:

/home/travis/build/Attnam/ivan/Main/Source/gods.cpp:1363:13: error: no member

      named 'LiquidRain' in 'lsquare'

    Square->LiquidRain(Beam, BLOOD);

    ~~~~~~  ^

1 error generated.

make[2]: *** [Main/CMakeFiles/ivan.dir/Source/godset.cpp.o] Error 1

make[1]: *** [Main/CMakeFiles/ivan.dir/all] Error 2

make: *** [all] Error 2

The command "make" exited with 2.
```

Reply:
Oops, I forgot I added LiquidRain in the Aslona branch. I'll update this with current master, then it should be OK.

Reply:
It's fixed by #576